spec.proto

```diff
@@ -311,6 +311,8 @@ message ProtoExecutionResult {
     string failureScreenshotFile = 11;
     /// Path to the screenshot files captured using Gauge screenshsot API.
     repeated string screenshotFiles = 12;
+    /// Flag to indicate skipping of the scenario execution
+    repeated bool skipScenario = 13;
```

Comment:
Why is this a repeated bool? This will translate to an array of Boolean at the language end.

Reply:
Yes, I've corrected this issue ...